src/common/libjob/job.h

```diff
@@ -35,6 +35,11 @@ enum job_lookup_flags {
      * - currently works for jobspec and R
      */
     FLUX_JOB_LOOKUP_JSON_DECODE = 1,
+    /* get current value of special fields by applying eventlog
+     * updates for those fields
+     * - currently works for jobspec and R
+     */
+    FLUX_JOB_LOOKUP_CURRENT = 2,
```

Comment:
A question about this commit. The comment here indicates "works for jobspec and R" but I only see R handled specifically in `lookup_current`. Maybe a comment in `lookup_current` as to why jobspec is not handled there would be helpful.

Reply:
ahh comment is probably a development history mistake, see #5635